### DISCUSSION TITLE
Synthesizer silent mode?

### DISCUSSION THREAD

**Discussion by tylerweston** [coqui-ai/TTS#1071] [General Q&A] [answered]
Hello! I was wondering if there is some way to suppress the output from the Synthesizer class. There doesn't appear to be any way to pass a "silent_mode" flag or anything like that and it appears to just straight up print stuff without checking any flags first anyways. Maybe there is some global option somewhere I'm missing? I guess I could redirect stdout but I'm wondering if there is a more official way first. Thank you.

**Comment by erogol** [comment, marked as answer]
Nope, there isn't a global option. Currently removing the print statement is the only way. Or you can pipe the command outputs to a file.

**Reply by tylerweston** [reply]
OK, thanks for the quick reply! I'm using rich console for all the other output in the program so hopefully attempting to pipe other things around doesn't mess anything up too much. Thanks for the wonderful project though! I've tried quite a few voice engines and this one has been giving me the best results so far. Cheers
